fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/impl/PropertiesRdfContext.java

```diff
@@ -56,19 +55,19 @@
      */
 
     public PropertiesRdfContext(final FedoraResource resource,
-                                final IdentifierConverter<Resource, FedoraResource> idTranslator)
-        throws RepositoryException {
+            final IdentifierConverter<Resource, FedoraResource> idTranslator)
+                    throws RepositoryException {
         super(resource, idTranslator);
         property2triple = new PropertyToTriple(resource.getNode().getSession(), idTranslator);
         concat(triplesFromProperties(resource()));
     }
 
     private Iterator<Triple> triplesFromProperties(final FedoraResource n)
-        throws RepositoryException {
+            throws RepositoryException {
         LOGGER.trace("Creating triples for node: {}", n);
         final Iterator<Property> allProperties = n.getNode().getProperties();
         final UnmodifiableIterator<Property> properties =
-            Iterators.filter(allProperties, not(isInternalProperty));
+                Iterators.filter(allProperties, not(isInternalProperty));
```

Comment:
Is this necessary?
Have you configured your eclipse with: https://github.com/fcrepo4/fcrepo4/tree/master/src/site/eclipse